pulse-rs/src/proplist.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright © 2017 Mozilla Foundation
+//
+// This program is made available under an ISC-style license.  See the
+// accompanying file LICENSE for details.
+
+use ffi;
+use std::ffi::CStr;
+use std::os::raw::c_char;
+
+#[derive(Debug)]
+pub struct Proplist(*mut ffi::pa_proplist);
+
+impl Proplist {
+    pub fn gets(&self, key: *const c_char) -> Option<&CStr> {
```

Comment:
Could this take a `&str` and avoid the caller having to use `cstr!` by converting it to a Cstr internally?